### PR TITLE
Fix create task

### PR DESCRIPTION
**Pull request by djino97**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/750?issue_count=2&issue_position=1&next_issue_id=639

### CODE REVIEW

**Review by TIove — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210707003534_ChangeTaskPropertiesTableColumn.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(ProjectServiceDbContext))]
+    [Migration("20210707003534_ChangeTaskPropertiesTableColumn")]
+    public class ChangeTaskPropertiesTableColumn : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```
Comment (BelehovEgor) [inline src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210707003534_ChangeTaskPropertiesTableColumn.cs:12]: Down?

File: src/ProjectService.Mappers/RequestsMappers/Interfaces/IDbTaskMapper.cs [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using System;
 
 namespace LT.DigitalOffice.ProjectService.Mappers.RequestsMappers.Interfaces
 {
     [AutoInject]
     public interface IDbTaskMapper
     {
-        public DbTask Map(CreateTaskRequest taskRequest);
+        public DbTask Map(CreateTaskRequest taskRequest, Guid authorId);
```
Comment (BelehovEgor) [inline src/ProjectService.Mappers/RequestsMappers/Interfaces/IDbTaskMapper.cs:11]: public in interface

**Review by BelehovEgor — APPROVED** [review]

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210707003534_ChangeTaskPropertiesTableColumn.cs (hunk 1 above)
Comment (djino97) [reply src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210707003534_ChangeTaskPropertiesTableColumn.cs:12]: Why?

File: src/ProjectService.Mappers/RequestsMappers/Interfaces/IDbTaskMapper.cs (hunk 2 above)
Comment (djino97) [reply src/ProjectService.Mappers/RequestsMappers/Interfaces/IDbTaskMapper.cs]: fixed

**Pull request merged**